src/libPMacc/include/communication/CommunicatorMPI.hpp

```diff
@@ -231,12 +229,12 @@ class CommunicatorMPI : public ICommunicator
     void exit()
```

Comment:
@psychocoderHPC we now call `MPI_Init`/`MPI_Finalize` in the main program(s). Should we actually remove the complete member here?